apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+require('babel-polyfill');
+window.React = require('react');
+window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
+var update = require('react-addons-update');
+
+window.TutorialExplorerManager = function (options) {
+  this.options = options;
+  var self = this;
+
+  const FilterChoice = React.createClass({
+    propTypes: {
+      onUserInput: React.PropTypes.func,
+      groupName: React.PropTypes.string,
+      name: React.PropTypes.string,
+      isCheckedInput: React.PropTypes.bool,
+      value: React.PropTypes.string,
+      selected: React.PropTypes.bool,
+      text: React.PropTypes.string
+    },
+
+    handleChange: function () {
+      this.props.onUserInput(
+        this.props.groupName,
+        this.props.name,
+        this.refs.isCheckedInput.checked
+      );
+    },
+
+    render() {
+      return (
+        <div  style={{userSelect: 'none', WebkitUserSelect: 'none', MozUserSelect: 'none', MsUserSelect: 'none'}}>
+          <label style={{fontFamily: "\"Gotham 4r\", sans-serif", fontSize: 13, paddingBottom: 0, marginBottom: 0, cursor: 'pointer'}}>
+            <input
+              type="checkbox"
+              value={this.props.value}
+              checked={this.props.selected}
+              ref="isCheckedInput"
+              onChange={this.handleChange}
+              style={{marginRight: 5}}
+            />
+            {this.props.text}
+          </label>
+        </div>
+      );
+    }
+  });
+
+  const FilterGroup = React.createClass({
+    propTypes: {
+      name: React.PropTypes.string,
+      text: React.PropTypes.string,
+      filterEntries: React.PropTypes.array,
+      selection: React.PropTypes.array,
+      onUserInput: React.PropTypes.func
+    },
+
+    render() {
+      return (
+        <div style={{paddingTop: 20, paddingRight: 40}}>
+          <div style={{fontFamily: '"Gotham 5r", sans-serif', borderBottom: 'solid grey 1px'}}>
+            {this.props.text}
+          </div>
+          {this.props.filterEntries.map(item => <FilterChoice groupName={this.props.name} name={item.name} text={item.text} selected={this.props.selection && this.props.selection.includes(item.name)} onUserInput={this.props.onUserInput} key={item.name}/>)}
+        </div>
+      );
+    }
+  });
+
+  const FilterSet = React.createClass({
+    propTypes: {
+      filterGroups: React.PropTypes.array,
+      onUserInput: React.PropTypes.func,
+      selection: React.PropTypes.object
+    },
+
+    render() {
+      return (
+        <div>
+          <div className="col-20">
+            <div style={{fontSize: 16}}>
+              Filter By
```

Comment:
Yes.  Since we rely on translations being provided by Pegasus, I was going to pass the strings in when initializing the TutorialExplorer, but in a later PR.